### ISSUE TITLE
Docs Update -- Contact/Help

### ISSUE THREAD

**Issue by andrewvyrros** [opendp/opendp#273]
In the nav-bar, we have a section `Contact` with links for people to contact us, get help, etc. This could use a cleanup pass. Some things to figure out:

* What's the purpose of this section?
* What should be the label?
* Should we merge the GitHub and Twitter icons on the right, or keep them separate?

Some similar projects to consult to see how they structure things:

* https://fairlearn.org
* https://pandas.pydata.org
* https://scikit-learn.org/stable/
* https://www.tensorflow.org
* https://pytorch.org
* https://www.rust-lang.org

**Comment by pdurbin** [comment]
@raprasad just mentioned that if we change the URL from https://docs.opendp.org/en/stable/contact/index.html to please let him know so he can make the adjustment in DP Creator.

**Comment by pdurbin** [comment]
I just made pull request #277 as a minimal clean up pass but I'll comment below on specific questions @andrewvyrros has raised:

> What's the purpose of this section?

I'm influenced by https://dataverse.org/contact where a variety of ways to contact the project are presented, like this:

![Screen Shot 2021-09-09 at 3 15 42 PM](https://user-images.githubusercontent.com/21006/132748792-6214e3b5-60ab-4512-8107-e19d453deb95.png)

The idea is that somebody wants to contact the project for one reason or another. They found a bug. They have a question. They want to discuss something on a forum. They want to discuss something privately.

In the pull request, I added a few more items (mailing list, issue tracker, security email) so that it looks like this:

![Screen Shot 2021-09-09 at 3 09 37 PM](https://user-images.githubusercontent.com/21006/132749262-f3952571-05c6-48dc-80a3-d06f30344805.png)

> What should be the label?

If we're fine with how things are, "Contact" is fine. If we want something fairly different, we should pick an appropriate label like "Community" or whatever we're going for.

> Should we merge the GitHub and Twitter icons on the right, or keep them separate?

I think they're fine. They're small.

> Some similar projects to consult to see how they structure things:

I looked briefly at these with question "From the header can I tell how I'd contact people involved in the project?" Generally, it seems like these projects are coming more from the perspective of "Given how many people are asking for support, how can I get them talking to each other?" We're doing this with GitHub Discussions but at this early stage I think it's ok to promote the idea of reaching out to us (in addition to each other).

I hope this makes sense! I'm totally fine with a complete re-do of "Contact" into something else.
